toolkit/tools/internal/file/file_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package file
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(m *testing.M) {
+	logger.InitStderrLog()
+	retVal := m.Run()
+	os.Exit(retVal)
+}
+
+func testFileName(t *testing.T) string {
+	return filepath.Join(t.TempDir(), t.Name())
+}
+
+func TestRemoveFileIfExistsValid(t *testing.T) {
```

Comment:
Thanks for adding the tests! Could we also add a check that the removed file does not exist after calling `RemoveFileIfExists`?

Reply:
Realized I wasn't using t.TempDir right, fixed that as well.